en/micro-integrator/docs/setup/deployment/k8s-samples/jms-sender-receiver.md

```diff
@@ -137,8 +153,45 @@ Follow the steps given below.
                 </td>
             </tr>
         </table>
+
+    3.  This step is only required if you already have a Docker image (in your local Docker repository) with the same name as the base image specified above. 
     
-    3.  Open the **integration_cr.yaml** file inside the Kubernetes project and add the environment variables as shown below. These values will be injected to the parameters defined in the proxy service.
+        !!! Info
+            In this scenario, WSO2 Integration Studio will first check if there is a difference in the two images before pulling the image specified in the **Base Image Repository** field. If the given base image is more updated, the existing image will be overwritten by this new image. Therefore, if you are currently using an older version, or if you have custom changes in your existing image, they will be replaced. 
+
+        To avoid your existing custom/older images from being replaced, add the following property under **dockerfile-maven-plugin -> executions -> execution -> configurations** in the `pom.xml` file of your Kubernetes Exporter project. This configuration will ensure that the base image will not be pulled when a Docker image already exists with the same name.
+
+        ```xml
+        <pullNewerImage>false</pullNewerImage>
+        ```
+
+        Example plugin configuration after adding the property:
+        ```xml
+          <plugin>
+            <groupId>com.spotify</groupId>
+            <artifactId>dockerfile-maven-plugin</artifactId>
+            <version>1.4.3</version>
+            <extensions>true</extensions>
+            <executions>
+              <execution>
+                <goals>
+                  <goal>build</goal>
+                  <goal>push</goal>
+                </goals>
+                <configuration>
+                  <username>${username}</username>
+                  <password>${password}</password>
+                  <repository>docker/helloworld</repository>
```

Comment:
How should we change this?
not helloworld yeah?

Reply:
Usually we dont have to change the docker repository for each project. So lets have a generic name like docker/samples.